plugin-config.yml

```diff
@@ -0,0 +1,41 @@
+redis_queue:
```

Comment:
Is this a duplicate to the `plugin-config.yml` in the `aries-backchannel/acapy` folder?  Is it needed?

Reply:
I'm not sure. there is a `plugin-config.template` file in `aries_backchannel/acapy` but i'm not sure how that get's put into the container for runtime.